src/command/list/human.rs

```diff
@@ -0,0 +1,1016 @@
+//! Define the "plain" format style for list commands.
```

Comment:
Should be "human" for this file :smiley:

```suggestion
//! Define the "human" format style for list commands.
```